spec/controllers/term_controller_spec.rb

```diff
@@ -5,7 +5,6 @@
 
   describe "GET term" do
     it "gets term by id " do
-      term = FactoryGirl.create(:term)
       get :show, id: term
```

Comment:
term is created in the let statement so this was repetition, just thought i'd clean it up as i noticed it :)